com.unity.render-pipelines.high-definition/Runtime/Lighting/VolumetricLighting/HDRenderPipeline.VolumetricClouds.cs

```diff
@@ -295,7 +295,10 @@ void UpdateShaderVariableslClouds(ref ShaderVariablesClouds cb, HDCamera hdCamer
                 cb._SunRight = currentSun.transform.right;
                 cb._SunUp = currentSun.transform.up;
 
-                cb._SunLightColor = m_lightList.directionalLights[0].color;
+                if (!shadowPass)
+                {
+                    cb._SunLightColor = m_lightList.directionalLights[0].color;
```

Comment:
We should still init the value to something no?

Reply:
in case we don't enter in the bracnh, will it be 0?

Reply:
If it is the shadow pass, the value by default is 0 yes